### PR TITLE
Bulgarian translation update

### PR DESCRIPTION
**Pull request by Syndamia**

### CODE REVIEW

File: localization/languages/bg.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "goForward": "Отиване една страница напред",
         "inspectElement": "Изследване на елемента",
         /* searchbar */
-        "pasteAndGo": null, //missing translation
+        "pasteAndGo": "Поставяне и посещаване", //missing translation
```
Comment (PalmerAL) [inline localization/languages/bg.json:30]: Can you remove the "missing translation" comments?

Comment (Syndamia) [reply localization/languages/bg.json:30]: Oh yeah, I completely forgot to remove them, give me 5 min

Comment (Syndamia) [reply localization/languages/bg.json]: Done

**Comment by PalmerAL** [comment]
Thanks!

**Pull request merged**